.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.rulers": [80, 100],
+}
```

Comment:
It seems like vscode settings file. Is there any policy about IDE or editor configuration? cc @planetarium/9c-dev  @planetarium/9c-backend